docs/help/Contents/Administration/admin_set_stmt.md

```diff
@@ -0,0 +1,31 @@
+# ADMIN SET REPLICA STATUS
+## description
+
+    该语句用于设置指定副本的状态。
+    该命令目前仅用于手动将某些副本状态设置为 BAD，从而使得系统能够自动修复这些副本。
+
+    语法：
+
+        ADMIN SET REPLICA STATUS
+        PROPERTIES ("key" = "value", ...);
+
+        目前支持如下属性：
+        "tablet_id"：必需。指定一个或多个 tablet id，逗号分隔。
+        "backend_id"：必需。指定 Backend Id.
+        "status"：必需。指定状态。当前仅支持 "bad"
+
+        如果指定的副本不存在，或状态已经是 bad，则会被忽略。
+
+## example
+
+    1. 设置 tablet 10003 在 BE 10001 上的副本状态为 bad。
+
+        ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "10003", "backend_id" = "10001", "status" = "bad");
+
+    1. 设置 tablet 10003,10004,10005 在 BE 10001 上的副本状态为 bad。
```

Comment:
Is there any chance to set the replica back to normal?
Or how is the whole process of the operation?

Reply:
No, it can't. If a replica is set to bad, it will be deleted anyway.